### PR TITLE
[chores] Added GitHub workflow for CI

### PR DESCRIPTION
**Pull request by pandafy**
- Removed Travis

----------------

*Blockers*
- [x] https://github.com/openwisp/openwisp-controller/pull/330
- [x] Add `GALAXY_TOKEN` secret @nemesisdesign

### CODE REVIEW

**Comment by atb00ker** [comment]
> Add GALAXY_TOKEN secret @nemesisdesign

Interesting, do we want to update from CI?
Because think it's only updated on release! :smile:

**Comment by pandafy** [comment]
> > Add GALAXY_TOKEN secret @nemesisdesign
> 
> Interesting, do we want to update from CI?
> Because think it's only updated on release!

@atb00ker I added this because it was already present in `.travis.yml`

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+name: OpenWISP CI Build
```
Comment (nemesisdesign) [inline .github/workflows/ci.yml:2]: let's change the name of this to `Ansible OpenWISP 2 CI Build`

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
This should be unblocked and you can complete it.

Screenshot from 2020-12-01 16-49-57

**Comment by pandafy** [comment]
The build passed with commits on #206 : https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html#github-actions-gotcha

**Comment by pandafy** [comment]
The build stalled at `TASK [ansible-openwisp2 : Install openwisp2 controller and its dependencies] ***` without pinning `channels_redis` dependency. Since it is something which has been already taken care of in #206, let's merge this and move forward with rest of the work. :smile:

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**